File: talleres c++/leydeomh.cpp
#include <iostream>
using namespace std;

int main() {
    int Voltaje, Resistencia;
    float Intensidad;  // Utilizamos float para que la intensidad sea un número decimal

    // Solicitar al usuario los valores del voltaje y la resistencia
    cout << "Ingrese el valor del voltaje del circuito: ";
    cin >> Voltaje;

    cout << "Ingrese el valor de la resistencia del circuito: ";
    cin >> Resistencia;

    // Calcular la intensidad (amperaje)
    Intensidad = static_cast<float>(Voltaje) / Resistencia;

    // Mostrar el resultado
    cout << "Al conectar un resistor de R " << Resistencia << " ohm a una fuente de V " << Voltaje
         << " voltaje circulará una corriente de " << Intensidad << " amperios" << endl;

    return 0;
}

File: talleres java/leydeomh.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Solicitar el voltaje y la resistencia del usuario
        System.out.print("Ingrese el valor del voltaje del circuito: ");
        int voltaje = scanner.nextInt();

        System.out.print("Ingrese el valor de la resistencia del circuito: ");
        int resistencia = scanner.nextInt();

        // Calcular la intensidad (amperaje)
        double intensidad = (double) voltaje / resistencia;

        // Mostrar el resultado
        System.out.println("Al conectar un resistor de R " + resistencia + " ohm a una fuente de V " + voltaje + " voltios, circulará una corriente de " + intensidad + " amperios.");

        scanner.close();
    }
}